addons/locales.sh

```diff
@@ -5,11 +5,8 @@
 # shellcheck disable=2034,2059,1091
 true
 SCRIPT_NAME="Locales"
-
 # shellcheck source=lib.sh
-source /var/scripts/fetch_lib.sh &>/dev/null || . <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh) &>/dev/null
-
```

Comment:
1. I think the whitespace around here isn't needed

Reply:
Yeah whatever. :)